lib/ci-helper.ts

```diff
@@ -11,6 +11,7 @@ import { MailArchiveGitHelper } from "./mail-archive-helper";
 import { MailCommitMapping } from "./mail-commit-mapping";
 import { IMailMetadata } from "./mail-metadata";
 import { IPatchSeriesMetadata } from "./patch-series-metadata";
+import { getPullRequesKeyFromtURL } from "./pullRequestKey";
```

Comment:
Typo? I think the `t` before `URL` belongs before the `Key` instead.

Reply:
Sorry, I usually catch that sort of thing.